src/burnchains/mod.rs

```diff
@@ -77,7 +74,7 @@ impl_array_hexstring_fmt!(Txid);
 impl_byte_array_newtype!(Txid, u8, 32);
 pub const TXID_ENCODED_SIZE: u32 = 32;
 
-#[derive(Serialize, Deserialize)]
+#[derive(Default, Serialize, Deserialize)]
```

Comment:
Please remove `Default`.  It's better to have an explicit constructor for a default value, because then it will stay the same even if Rust's behavior changes.

Reply:
Address with latest